modules/openapi-generator/src/main/resources/Javascript/es6/ApiClient.mustache

```diff
@@ -469,10 +469,12 @@ class ApiClient {
             request.end((error, response) => {
                 if (error) {
                     var err = {};
-                    err.status = response.status;
-                    err.statusText = response.statusText;
-                    err.body = response.body;
-                    err.response = response;
+                    if(response && response.status && response.statusText && response.body){
```

Comment:
@macjohnny I think it's better check all the attributes that will be assigned to `err`.

Reply:
I think just checking `if (response)` is better, since it avoids null-pointers and still assigns e.g. `err.status` even if e.g. `response.body` is empty